src/components/TaskCard/TaskCard.scss

```diff
@@ -30,4 +30,9 @@
         padding: 0.5rem;
         background-color: base.$bg-gray-50;
     }
+
+    &--complete &__title {
+        text-decoration: line-through;
+        color: #0bc375;
```

Comment:
If you are maintaining color variable then put this on that file also. Thank you!